app-config.yaml

```diff
@@ -293,6 +293,14 @@ dynatrace:
 # Plugin: Permission
 permission:
   enabled: ${PERMISSION_ENABLED}
+  rbac:
+    admin:
+      users:
+        - name: user:default/alice
+        - name: group:default/admins
+    policies-csv-file: ./rbac/rbac-policy.csv
+    database:
+      enabled: true
```

Comment:
This should be set to false.  Since we are using the policy file there is no need to store the policies in the database

Reply:
Done